test/Teepee.js

```diff
@@ -2498,6 +2498,16 @@ describe('Teepee', () => {
       expect(subsidiary.numRetries, 'to equal', 99);
     });
 
+    it('should not throw unhandled error when emitting error from subsidiary object', () => {
+      const teepee = new Teepee();
+      const subsidiary = teepee.subsidiary();
+      const cb = sinon.spy();
+      subsidiary.on('error', cb);
+      subsidiary.emit('error');
+
+      expect(cb.called, 'to be true');
```

Comment:
How many times do you expect `cb` to be called?

Reply:
Should be twice because we are attaching the cb to both subsidiary client and base client to avoid unhandled error.